pom.xml

```diff
@@ -29,8 +29,8 @@
                     <compilerArgument>-Xlint:deprecation</compilerArgument>
                     -->
                     <fork>true</fork>
-                    <source>1.8</source>
```

Comment:
1.8 --> 14?

Reply:
```Java versions before 9 simply had a different naming scheme. So, Java 8 can also be called 1.8, Java 5 can be called 1.5, etc.```

https://dzone.com/articles/a-guide-to-java-versions-and-features